src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -44,6 +45,7 @@ class PressableWithSecondaryInteraction extends Component {
                 delayLongPress={200}
                 onLongPress={this.props.onSecondaryInteraction}
                 onPressOut={this.props.onPressOut}
+                onPress={this.props.onPress}
```

Comment:
If you're adding an `onPress`, you need to:

1. Update the `propTypes`
1. Include the `onPress` prop in the native implementation.